frontends/concrete-python/concrete/fhe/tfhers/context.py

```diff
@@ -0,0 +1,192 @@
+"""
+Declaration of `tfhers.Context` class.
+"""
+
+from typing import List, Optional, Union
+
+from concrete.compiler import TfhersExporter, TfhersFheIntDescription
+
+from concrete import fhe
+
+from .dtypes import EncryptionKeyChoice, TFHERSIntegerType
+
+
+class Context:
+    """TFHErs Context extend a Circuit with TFHErs functionalities."""
+
+    circuit: "fhe.Circuit"
```

Comment:
why it's between quotes?

Reply:
Probably because using the proper type would introduce circular import dependencies. Python 🙄 ...

Reply:
Exactly @aPere3